### PR TITLE
Add support for listing runbooks

### PR DESCRIPTION
**Pull request by ejadib**

### CODE REVIEW

File: AzureBot/Dialogs/ActionDialog.cs [hunk 1]
```diff
@@ -193,6 +193,43 @@ public class ActionDialog : LuisDialog<string>
...
+            if (string.IsNullOrEmpty(accessToken))
+            {
+                return;
+            }
+
+            var subscriptionId = context.GetSubscriptionId();
+
+            var automationAccounts = await new AzureRepository().ListRunbooksAsync(accessToken, subscriptionId);
+
+            if (automationAccounts.Any())
```
Comment (dtzar) [inline AzureBot/Dialogs/ActionDialog.cs]: We should detect if there is only one and then the output will look different versus more than one.

File: AzureBot/Dialogs/ActionDialog.cs [hunk 2]
```diff
@@ -193,6 +193,43 @@ public class ActionDialog : LuisDialog<string>
...
+            {
+                var runbooksText = automationAccounts.Aggregate(
+                     string.Empty,
+                    (current, next) =>
+                    {
+                        return current += next.Runbooks.Aggregate(
+                            string.Empty,
+                            (currentRunbooks, nextRunbook) => 
+                            {
+                                return currentRunbooks += $"\n\r• {nextRunbook.RunbookName} ({next.AutomationAccountName})";
```
Comment (dtzar) [inline AzureBot/Dialogs/ActionDialog.cs]: If only one Automation account, there is no need to put the Automation account name next to it, we should put "listing all runbooks from _automationaccountname_:"

**Comment by dtzar** [comment]
If we have more than one automation account, I like the structured output list better:
Automation Account 1
- Runbook A
- Runbook B
- Runbbok C
Automation Account 2
- Runbok X

File: AzureBot/Dialogs/ActionDialog.cs (hunk 2 above)
Comment (ejadib) [inline AzureBot/Dialogs/ActionDialog.cs]: done

**Pull request merged**